app/controllers/provider_interface/decisions_controller.rb

```diff
@@ -1,9 +1,39 @@
 module ProviderInterface
   class DecisionsController < ProviderInterfaceController
     before_action :set_application_choice
+    before_action :confirm_application_is_in_decision_pending_state, only: %i[new create]
     before_action :requires_make_decisions_permission
 
+    def new
+      @wizard = OfferWizard.new(offer_store,
+                                offer_context_params(@application_choice.course_option).merge!(current_step: 'select_option'))
+      @wizard.save_state!
+    end
+
+    def create
+      @wizard = OfferWizard.new(offer_store, { decision: selected_decision })
+
+      if @wizard.valid_for_current_step?
+
+        @wizard.save_state!
+
+        if @wizard.decision == 'rejection'
+          @wizard.clear_state!
```

Comment:
Clearing the state here is causing errors if I go back in the browser and try to make an offer again/choose rejection, is there a way we could avoid this?

Reply:
This is interesting. I guess we can move clearing the wizard to the end of the rejection process, as that would be the end of the decision flow when rejection is picked.

Reply:
Moved to the end of the rejection process, which seems to have resolved this issue.